inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -690,6 +743,11 @@ layout layout_optimizer::get_expected_layout(layout const& current_layout,
 
     auto output_layout = node.calc_output_layout();
 
+    // bool valid_onednn_post_ops = get_post_ops_count(node) <= 32;
```

Comment:
Isn't it necessary to check `get_post_ops_count`? If so we may remove the get_post_ops_count function itself?

Reply:
I'm thinking about it.

Reply:
Deleted the comment. But get_post_ops_count function was left due to potential limitations (10 bin post-ops) in some onednn primitives (conv, deconv, matmul supports upto 32 bin post-ops, but others need extra checkings).